addCRH.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
```

Comment:
You may want to add a copyright to this very file as well :)
Please, feel free to add your name as the contributor